runtime-sdk/src/modules/consensus/mod.rs

```diff
@@ -0,0 +1,279 @@
+//! Consensus module.
```

Comment:
Maybe say something like "Module for communicating with the consensus layer."? The module could also be called `consensus_msgs` or something?

Reply:
Hm yeah a more detailed doc comment is on the module struct itself. I'll move it up here and update it a bit.

Reply:
About the module name, i think just `consensus` makes more sense than `consensus_msgs` as the module also contains methods for querying consensus state (currently only `account` though), and some other helper methods such as `consensus_denomination` and `ensure_compatible_tx_signer`.

Reply:
Makes sense.